core/spree_core.gemspec

```diff
@@ -30,7 +30,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'json', '~> 1.7'
   s.add_dependency 'kaminari', '~> 0.15', '>= 0.15.1'
   s.add_dependency 'monetize', '~> 1.1'
-  s.add_dependency 'paperclip', '~> 4.2.0'
+  s.add_dependency 'paperclip'
```

Comment:
Hi @sterrym - don't do multiple things in one PR - it's nearly impossible to merge something like this